File: 18.py
'''s="Backbanchers"
print(type(s))
for i in s:
    print("."*i+s[i:])
'''
def triDownwards(S):
        s=""
        for i in range(0,len(S)):
            s+="."*i+S[i:]
        return s

s="Backbanchers"
print(triDownwards(s))

File: 18.c
class Solution:
	def numberPattern(self, n):
		l=[]
		for i in range(1,n+1):
		    s=""
		    for x in range(1,i+1):
		        s+=str(x)
		    for y in range(i-1,0,-1):
		        s+=str(y)
		    l.append(s)
        return l
